{"repository":"https://github.com/stdm/zhaw_deep_voice","dockerfilePath":"Dockerfile","startTime":1672541421148,"clone":{"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/stdm/zhaw_deep_voice'...\n","error":null,"commit":"197ee168f2b4e806cc2e6e030579e8085186e443"},"originalBuild":{"startTimestamp":1672541422230,"endTimestamp":1672541670618,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t zhaw_deep_voice:latest -f /tmp/dinghy-analysis/stdm/zhaw_deep_voice/Dockerfile ."},"stdout":"Sending build context to Docker daemon  636.9kB\r\r\nStep 1/12 : FROM tensorflow/tensorflow:1.13.1-gpu-py3\n1.13.1-gpu-py3: Pulling from tensorflow/tensorflow\n34667c7e4631: Pulling fs layer\nd18d76a881a4: Pulling fs layer\n119c7358fbfc: Pulling fs layer\n2aaf13f3eff0: Pulling fs layer\n643564d518c8: Pulling fs layer\n1fea03e629a4: Pulling fs layer\n45402f4cf61d: Pulling fs layer\n45f7c407b07b: Pulling fs layer\n00e5163fe3e0: Pulling fs layer\n7d071071ef98: Pulling fs layer\n5119bdada1e4: Pulling fs layer\n64a9355aa772: Pulling fs layer\n21f5ce47fe21: Pulling fs layer\n5566cd4bac12: Pulling fs layer\n2aaf13f3eff0: Waiting\n58c608a4c711: Pulling fs layer\n643564d518c8: Waiting\n7d071071ef98: Waiting\n5119bdada1e4: Waiting\n1fea03e629a4: Waiting\n64a9355aa772: Waiting\n00e5163fe3e0: Waiting\n45402f4cf61d: Waiting\n21f5ce47fe21: Waiting\n5566cd4bac12: Waiting\n58c608a4c711: Waiting\n45f7c407b07b: Waiting\n119c7358fbfc: Verifying Checksum\n119c7358fbfc: Download complete\nd18d76a881a4: Download complete\n2aaf13f3eff0: Verifying Checksum\n2aaf13f3eff0: Download complete\n34667c7e4631: Verifying Checksum\n34667c7e4631: Download complete\n643564d518c8: Verifying Checksum\n643564d518c8: Download complete\n1fea03e629a4: Verifying Checksum\n1fea03e629a4: Download complete\n45402f4cf61d: Verifying Checksum\n45402f4cf61d: Download complete\n7d071071ef98: Verifying Checksum\n7d071071ef98: Download complete\n34667c7e4631: Pull complete\n5119bdada1e4: Verifying Checksum\n5119bdada1e4: Download complete\nd18d76a881a4: Pull complete\n119c7358fbfc: Pull complete\n2aaf13f3eff0: Pull complete\n64a9355aa772: Verifying Checksum\n64a9355aa772: Download complete\n00e5163fe3e0: Verifying Checksum\n00e5163fe3e0: Download complete\n643564d518c8: Pull complete\n5566cd4bac12: Verifying Checksum\n5566cd4bac12: Download complete\n58c608a4c711: Verifying Checksum\n58c608a4c711: Download complete\n1fea03e629a4: Pull complete\n45402f4cf61d: Pull complete\n21f5ce47fe21: Verifying Checksum\n21f5ce47fe21: Download complete\n45f7c407b07b: Verifying Checksum\n45f7c407b07b: Download complete\n45f7c407b07b: Pull complete\n00e5163fe3e0: Pull complete\n7d071071ef98: Pull complete\n5119bdada1e4: Pull complete\n64a9355aa772: Pull complete\n21f5ce47fe21: Pull complete\n5566cd4bac12: Pull complete\n58c608a4c711: Pull complete\nDigest: sha256:0f949ccc690d9c50e9b46b16d9030c2c6845af621c2e7e82c4bf59803edc69b5\nStatus: Downloaded newer image for tensorflow/tensorflow:1.13.1-gpu-py3\n ---> 20a4b7fa03e7\nStep 2/12 : RUN pip install --upgrade pip\n ---> Running in 804eee1fc227\nCollecting pip\n  Downloading https://files.pythonhosted.org/packages/27/79/8a850fe3496446ff0d584327ae44e7500daf6764ca1a382d2d02789accf7/pip-20.3.4-py2.py3-none-any.whl (1.5MB)\nInstalling collected packages: pip\n  Found existing installation: pip 19.1\n    Uninstalling pip-19.1:\n      Successfully uninstalled pip-19.1\nSuccessfully installed pip-20.3.4\nRemoving intermediate container 804eee1fc227\n ---> e65ce5fd5ee8\nStep 3/12 : RUN pip install numpy==1.14.5\n ---> Running in 2b2b658e6de8\n\u001b[91mDEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.\n\u001b[0mCollecting numpy==1.14.5\n  Downloading numpy-1.14.5-cp35-cp35m-manylinux1_x86_64.whl (12.1 MB)\nInstalling collected packages: numpy\n  Attempting uninstall: numpy\n    Found existing installation: numpy 1.16.3\n    Uninstalling numpy-1.16.3:\n      Successfully uninstalled numpy-1.16.3\nSuccessfully installed numpy-1.14.5\nRemoving intermediate container 2b2b658e6de8\n ---> 6f14f9f8a552\nStep 4/12 : RUN pip install scipy==0.19.1\n ---> Running in 9e1b8fa6cef2\n\u001b[91mDEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.\n\u001b[0mCollecting scipy==0.19.1\n  Downloading scipy-0.19.1-cp35-cp35m-manylinux1_x86_64.whl (47.9 MB)\nRequirement already satisfied: numpy>=1.8.2 in /usr/local/lib/python3.5/dist-packages (from scipy==0.19.1) (1.14.5)\nInstalling collected packages: scipy\nSuccessfully installed scipy-0.19.1\nRemoving intermediate container 9e1b8fa6cef2\n ---> d2bcf76c085b\nStep 5/12 : RUN pip install scikit-learn==0.19.1\n ---> Running in 088a7bde4076\n\u001b[91mDEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.\n\u001b[0mCollecting scikit-learn==0.19.1\n  Downloading scikit_learn-0.19.1-cp35-cp35m-manylinux1_x86_64.whl (12.2 MB)\nInstalling collected packages: scikit-learn\nSuccessfully installed scikit-learn-0.19.1\nRemoving intermediate container 088a7bde4076\n ---> 45e0bf109ad7\nStep 6/12 : RUN pip install matplotlib==3.0.3\n ---> Running in 4b409180ab6e\n\u001b[91mDEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.\n\u001b[0mCollecting matplotlib==3.0.3\n  Downloading matplotlib-3.0.3-cp35-cp35m-manylinux1_x86_64.whl (13.0 MB)\nRequirement already satisfied: numpy>=1.10.0 in /usr/local/lib/python3.5/dist-packages (from matplotlib==3.0.3) (1.14.5)\nCollecting python-dateutil>=2.1\n  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)\nCollecting cycler>=0.10\n  Downloading cycler-0.10.0-py2.py3-none-any.whl (6.5 kB)\nCollecting pyparsing!=2.0.4,!=2.1.2,!=2.1.6,>=2.0.1\n  Downloading pyparsing-2.4.7-py2.py3-none-any.whl (67 kB)\nCollecting kiwisolver>=1.0.1\n  Downloading kiwisolver-1.1.0-cp35-cp35m-manylinux1_x86_64.whl (90 kB)\nRequirement already satisfied: six in /usr/local/lib/python3.5/dist-packages (from cycler>=0.10->matplotlib==3.0.3) (1.12.0)\nRequirement already satisfied: setuptools in /usr/local/lib/python3.5/dist-packages (from kiwisolver>=1.0.1->matplotlib==3.0.3) (41.0.1)\nInstalling collected packages: python-dateutil, pyparsing, kiwisolver, cycler, matplotlib\nSuccessfully installed cycler-0.10.0 kiwisolver-1.1.0 matplotlib-3.0.3 pyparsing-2.4.7 python-dateutil-2.8.2\nRemoving intermediate container 4b409180ab6e\n ---> f5348ed61bca\nStep 7/12 : RUN pip install keras==2.2.4\n ---> Running in e2592ec89847\n\u001b[91mDEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.\n\u001b[0mCollecting keras==2.2.4\n  Downloading Keras-2.2.4-py2.py3-none-any.whl (312 kB)\nRequirement already satisfied: six>=1.9.0 in /usr/local/lib/python3.5/dist-packages (from keras==2.2.4) (1.12.0)\nRequirement already satisfied: scipy>=0.14 in /usr/local/lib/python3.5/dist-packages (from keras==2.2.4) (0.19.1)\nCollecting pyyaml\n  Downloading PyYAML-5.3.1.tar.gz (269 kB)\nRequirement already satisfied: numpy>=1.9.1 in /usr/local/lib/python3.5/dist-packages (from keras==2.2.4) (1.14.5)\nRequirement already satisfied: keras-applications>=1.0.6 in /usr/local/lib/python3.5/dist-packages (from keras==2.2.4) (1.0.7)\nRequirement already satisfied: keras-preprocessing>=1.0.5 in /usr/local/lib/python3.5/dist-packages (from keras==2.2.4) (1.0.9)\nRequirement already satisfied: h5py in /usr/local/lib/python3.5/dist-packages (from keras==2.2.4) (2.9.0)\nBuilding wheels for collected packages: pyyaml\n  Building wheel for pyyaml (setup.py): started\n  Building wheel for pyyaml (setup.py): finished with status 'done'\n  Created wheel for pyyaml: filename=PyYAML-5.3.1-cp35-cp35m-linux_x86_64.whl size=45922 sha256=519052af8c7f4eb441af7d6b1be8c0a7872b4169a0bc84eeb8af5c433a9cde0d\n  Stored in directory: /root/.cache/pip/wheels/57/d0/2c/e2003abb5bc1a94c2e8a6fe1c03b8055d074e34c13672e7eb7\nSuccessfully built pyyaml\nInstalling collected packages: pyyaml, keras\nSuccessfully installed keras-2.2.4 pyyaml-5.3.1\nRemoving intermediate container e2592ec89847\n ---> 2e2ee7e78d05\nStep 8/12 : RUN pip install librosa==0.5.1\n ---> Running in 9b2cf716dd44\n\u001b[91mDEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.\n\u001b[0mCollecting librosa==0.5.1\n  Downloading librosa-0.5.1.tar.gz (1.5 MB)\nCollecting audioread>=2.0.0\n  Downloading audioread-2.1.9.tar.gz (377 kB)\nRequirement already satisfied: numpy>=1.8.0 in /usr/local/lib/python3.5/dist-packages (from librosa==0.5.1) (1.14.5)\nRequirement already satisfied: scipy>=0.13.0 in /usr/local/lib/python3.5/dist-packages (from librosa==0.5.1) (0.19.1)\nRequirement already satisfied: scikit-learn>=0.14.0 in /usr/local/lib/python3.5/dist-packages (from librosa==0.5.1) (0.19.1)\nCollecting joblib>=0.7.0\n  Downloading joblib-0.14.1-py2.py3-none-any.whl (294 kB)\nCollecting decorator>=3.0.0\n  Downloading decorator-5.1.1-py3-none-any.whl (9.1 kB)\nRequirement already satisfied: six>=1.3 in /usr/local/lib/python3.5/dist-packages (from librosa==0.5.1) (1.12.0)\nCollecting resampy>=0.1.2\n  Downloading resampy-0.4.2-py3-none-any.whl (3.1 MB)\n  Downloading resampy-0.4.1-py3-none-any.whl (3.1 MB)\n  Downloading resampy-0.4.0-py3-none-any.whl (3.1 MB)\n  Downloading resampy-0.3.1-py3-none-any.whl (3.1 MB)\nCollecting numba>=0.47\n  Downloading numba-0.47.0-cp35-cp35m-manylinux1_x86_64.whl (3.7 MB)\nCollecting numpy>=1.8.0\n  Downloading numpy-1.18.5-cp35-cp35m-manylinux1_x86_64.whl (19.9 MB)\nCollecting llvmlite>=0.31.0dev0\n  Downloading llvmlite-0.32.1.tar.gz (104 kB)\nRequirement already satisfied: setuptools in /usr/local/lib/python3.5/dist-packages (from numba>=0.47->resampy>=0.1.2->librosa==0.5.1) (41.0.1)\nBuilding wheels for collected packages: audioread, librosa, llvmlite\n  Building wheel for audioread (setup.py): started\n  Building wheel for audioread (setup.py): finished with status 'done'\n  Created wheel for audioread: filename=audioread-2.1.9-py3-none-any.whl size=26175 sha256=8073fe37c79d8b71a185d77a0b1d49e66b341e412a18e3a064295ce331df605c\n  Stored in directory: /root/.cache/pip/wheels/e8/c6/2d/1aed6b970f21374914e23b576ca11a295d2b445c3af1cc80f6\n  Building wheel for librosa (setup.py): started\n  Building wheel for librosa (setup.py): finished with status 'done'\n  Created wheel for librosa: filename=librosa-0.5.1-py3-none-any.whl size=1548711 sha256=0e0a348c33105fd18c9e55718dc348d17787becfa7a41d9467c9d61e83826ce1\n  Stored in directory: /root/.cache/pip/wheels/b9/32/15/604fa29fd49782ea613aa033c67378e5cff0edea572caf59a6\n  Building wheel for llvmlite (setup.py): started\n  Building wheel for llvmlite (setup.py): finished with status 'error'\n\u001b[91m  ERROR: Command errored out with exit status 1:\n   command: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' bdist_wheel -d /tmp/pip-wheel-fz2r0tj0\n       cwd: /tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/\n  Complete output (26 lines):\n  running bdist_wheel\n  /usr/bin/python3 /tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\n  LLVM version... Traceback (most recent call last):\n    File \"/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\", line 106, in main_posix\n      out = subprocess.check_output([llvm_config, '--version'])\n    File \"/usr/lib/python3.5/subprocess.py\", line 626, in check_output\n      **kwargs).stdout\n    File \"/usr/lib/python3.5/subprocess.py\", line 693, in run\n      with Popen(*popenargs, **kwargs) as process:\n    File \"/usr/lib/python3.5/subprocess.py\", line 947, in __init__\n      restore_signals, start_new_session)\n    File \"/usr/lib/python3.5/subprocess.py\", line 1551, in _execute_child\n      raise child_exception_type(errno_num, err_msg)\n  FileNotFoundError: [Errno 2] No such file or directory: 'llvm-config'\n  \n  During handling of the above exception, another exception occurred:\n  \n  Traceback (most recent call last):\n    File \"/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\", line 192, in <module>\n      main()\n    File \"/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\", line 182, in main\n      main_posix('linux', '.so')\n    File \"/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\", line 109, in main_posix\n      \"to the path for llvm-config\" % (llvm_config,))\n  RuntimeError: llvm-config failed executing, please point LLVM_CONFIG to the path for llvm-config\n  error: command '/usr/bin/python3' failed with exit status 1\n  ----------------------------------------\n\u001b[0m\u001b[91m  ERROR: Failed building wheel for llvmlite\n\u001b[0m  Running setup.py clean for llvmlite\nSuccessfully built audioread librosa\nFailed to build llvmlite\nInstalling collected packages: numpy, llvmlite, numba, resampy, joblib, decorator, audioread, librosa\n  Attempting uninstall: numpy\n    Found existing installation: numpy 1.14.5\n    Uninstalling numpy-1.14.5:\n      Successfully uninstalled numpy-1.14.5\n    Running setup.py install for llvmlite: started\n    Running setup.py install for llvmlite: finished with status 'error'\n\u001b[91m    ERROR: Command errored out with exit status 1:\n     command: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' install --record /tmp/pip-record-x8snpn03/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.5/llvmlite\n         cwd: /tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/\n    Complete output (29 lines):\n    running install\n    running build\n    got version from file /tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/llvmlite/_version.py {'full': 'aa11b129c0b55973067422397821ae6d44fa5e70', 'version': '0.32.1'}\n    running build_ext\n    /usr/bin/python3 /tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\n    LLVM version... Traceback (most recent call last):\n      File \"/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\", line 106, in main_posix\n        out = subprocess.check_output([llvm_config, '--version'])\n      File \"/usr/lib/python3.5/subprocess.py\", line 626, in check_output\n        **kwargs).stdout\n      File \"/usr/lib/python3.5/subprocess.py\", line 693, in run\n        with Popen(*popenargs, **kwargs) as process:\n      File \"/usr/lib/python3.5/subprocess.py\", line 947, in __init__\n        restore_signals, start_new_session)\n      File \"/usr/lib/python3.5/subprocess.py\", line 1551, in _execute_child\n        raise child_exception_type(errno_num, err_msg)\n    FileNotFoundError: [Errno 2] No such file or directory: 'llvm-config'\n    \n    During handling of the above exception, another exception occurred:\n    \n    Traceback (most recent call last):\n      File \"/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\", line 192, in <module>\n        main()\n      File \"/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\", line 182, in main\n        main_posix('linux', '.so')\n      File \"/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/ffi/build.py\", line 109, in main_posix\n        \"to the path for llvm-config\" % (llvm_config,))\n    RuntimeError: llvm-config failed executing, please point LLVM_CONFIG to the path for llvm-config\n    error: command '/usr/bin/python3' failed with exit status 1\n    ----------------------------------------\n\u001b[0m\u001b[91mERROR: Command errored out with exit status 1: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-iy_j8uyo/llvmlite_89e76c2b743a49df850a330cfe16a256/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' install --record /tmp/pip-record-x8snpn03/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.5/llvmlite Check the logs for full command output.\n\u001b[0mRemoving intermediate container 9b2cf716dd44\n","stderr":"The command '/bin/bash -c pip install librosa==0.5.1' returned a non-zero code: 1\n"},"endTime":1672541670686}